packages/glimmer-runtime/lib/modifier/interfaces.ts

```diff
@@ -4,15 +4,18 @@ import { DynamicScope } from '../environment';
 import { Destroyable } from 'glimmer-util';
 
 export interface ModifierManager<T> {
+  args: EvaluatedArgs;
+  dynamicScope: DynamicScope;
```

Comment:
This is incorrect. If you need them, you could save it inside the `T` you are creating, but the manager shouldn't have them. The manager is usually a singleton. (i.e. just ✂️  these two lines)